com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/persistence/DirectLinkingResourceStorageFacade.java

```diff
@@ -96,6 +96,11 @@ protected String computeOutputPath(final URI resourceUri) {
     return uri.deresolve(srcContainerURI, false, false, true).path();
   }
 
+  @Override

```

Comment:
this override isn't really needed?

Reply:
It will allow us to get rid of one override in ASMD.